cva6/tests/custom/sv32/vm_bare_mode_02.S

```diff
@@ -0,0 +1,147 @@
+#=======================================================================
+# SATP mode =Bare for Level 0 PTE 
+#-----------------------------------------------------------------------
+# Test Description:
+#
+# If PTE does not have the SATP setuped then, accessing it
+# would raise a ILLEGAL_INSTRUCTION fault exception.
+# When satp.mode=Bare and PTE has (r,w,x) PMP permissions, this test
+# covers the following scenarios in both supervisor and user privilege
+# modes for level1 PTE.
+#
+# Set SATP = Bare and rest bits to zeros and test the access to PTE.
+#=======================================================================
+
+#include "macros.h"
+
+#ifdef smode
+    #define SET_PTE_U 0
+#else
+    #define SET_PTE_U PTE_U
+#endif
+
+#define _MMODE_  "M"
+#define _SUMODE_ "SU"
+.set va, 0x82000000
+
+.text
+.global _start
+
+_start:
+    ALL_MEM_PMP                                                            # PMP permission to all the memory
+    la t1,trap_handler                                                     # loads the address of trap handler 
+    csrw mtvec,t1                                                          # sets the mtvec to trap handler 
+    
+# ----------------------Access test prolog------------------------------ 
+
+    TEST_PROLOG(check_access,CAUSE_ILLEGAL_INSTRUCTION)                    # load the addr and expected cause 
+
+# -------------LEVEL 1 PTE Setup for execute test------------------------
+                                                                           # Setup a new PTE to test execute
+    la a1,check_access                                                     # loads the address of label vm_en
+    la a0, va                                                              # set the VA to PA (identity mapping)
+    ori a2, x0, ( PTE_D | PTE_A | SET_PTE_U | PTE_X | PTE_V )              # sets the permission bits
+    PTE_SETUP_RV32(a1, a2, t1, a0, pgtb_l1, LEVEL1)                        # setup the PTE for level1
+
+# ----------------Set the SATP to Bare_Mode and change the mode---------------------
+
+    csrw satp, 0x04           	                                           # Write satp with no zero value (bare mode)
```

Comment:
satp should have the base address of page table

Reply:
Not possible the ppn[0]!=0 for the base address of page table